raiden/utils/__init__.py

```diff
@@ -159,7 +159,7 @@ def get_contract_path(contract_name: str) -> str:
     return get_relative_path(contract_path)
 
 
-def get_system_spec():
+def get_system_spec() -> typing.Optional[typing.Dict[str, str]]:
```

Comment:
This is never an `Optional`, just the key `raiden` might be optional, but I think we should fix that.